packages/mobile-top-nav/src/assets/img/search.js

```diff
@@ -1,15 +1,35 @@
-import { LitElement, html } from 'lit-element';
+import { LitElement, html, css } from 'lit-element';
+import Icon from './icon';
+
+class Search extends Icon {
+  constructor() {
+    super();
+    this.width = "27";
+    this.height = Math.ceil((+this.width) * 1.11);
+  }
 
-class Search extends LitElement {
   static get properties() {
-    return {
-      colour: { type: String }
-    };
+    const superclassProps = super.properties;
+    return Object.assign({
+      width: { type: String },
```

Comment:
Why is this a string?

Reply:
I set it up as a string because the attribute value comes through as a string, not thinking about how LitElement will attempt to coerce it to the type specified.